mifospay/src/main/java/org/mifos/mobilewallet/mifospay/savedcards/ui/CardsFragment.java

```diff
@@ -144,6 +144,27 @@ public void onItemLongPress(View childView, int position) {
                 }));
     }
 
+    private void deleteCard(final int position) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
```

Comment:
The code can be shortened.

Reply:
@luckyman20 what changes should i make ?

Reply:
Keep it continuous.

Reply:
@luckyman20 you mean without the function ?
If yes then that might lead to Travis build fail due to excessive size of a line of code, and adding so many indentation will look bad in my opinion.

Reply:
No, it should be in the function only. Example - `new AlertDialog.Builder(getContext()).setTitle(...).setPositiveButton().setNegativeButton()`

I hope you get my point.

Reply:
Yes i get it. I'll make the changes.